frontend/portals/MaintenanceMode/index.jsx

```diff
@@ -62,15 +72,39 @@ class MaintenanceMode extends Component {
     openPageExtern({ src: imageHref });
   };
 
+  /**
+   * Checks if the app version is blocked.
+   * @param {string} appVersion App version
+   * @returns {boolean}
+   */
+  appVersionIsBlocked = (appVersion) => {
+    const appVersions = this.props.isIosDevice ? iosAppVersions : androidAppVersions;
+
+    // All versions
+    if (!appVersions.length) {
+      return true;
+    }
+
+    // Client version
+    if (appVersions.includes(appVersion)) {
```

Comment:
since includes returns a boolean value you don't need the if statement you can just 
`return appVersions.includes(appVersion);`